src/modules/sensors/sensors.cpp

```diff
@@ -167,12 +167,12 @@ class Sensors : public ModuleBase<Sensors>
 	int		_vcontrol_mode_sub;		/**< vehicle control mode subscription */
 	int 		_params_sub;			/**< notification of parameter updates */
 
-	orb_advert_t	_sensor_pub;			/**< combined sensor data topic */
-	orb_advert_t	_battery_pub[BOARD_NUMBER_BRICKS] = {nullptr};			/**< battery status */
```

Comment:
+1 let's keep this change for consistency, then I'm happy to help uniformize the init style.